Ifpack2: Clang build error in test

## Bug Report
@trilinos/ifpack2 

### Description

```
.../packages/ifpack2/test/unit_tests/Ifpack2_UnitTestRelaxation.cpp:202:28: error: 'getLocalViewHost' following the 'template' keyword does not refer to a template
                y.template getLocalViewHost ().data ());
```

### Steps to Reproduce

Build with stock Clang on Mac laptop.

--- a/packages/ifpack2/test/unit_tests/Ifpack2_UnitTestRelaxation.cpp
+++ b/packages/ifpack2/test/unit_tests/Ifpack2_UnitTestRelaxation.cpp
@@ -199,7 +199,7 @@ TEUCHOS_UNIT_TEST_TEMPLATE_3_DECL(Ifpack2Relaxation, Test2, Scalar, LocalOrdinal
                 y.template getLocalView<Kokkos::HostSpace> ().data ());
 #else
   TEST_EQUALITY(x.getLocalViewHost ().data (),
-                y.template getLocalViewHost ().data ());
+                y.getLocalViewHost ().data ());
 #endif
 
   prec.apply(x, y);

